google/cloud/bigtable/internal/logging_data_client.cc

```diff
@@ -140,6 +140,15 @@ LoggingDataClient::AsyncSampleRowKeys(
   return child_->AsyncSampleRowKeys(context, request, cq, tag);
 }
 
+std::unique_ptr<::grpc::ClientAsyncReaderInterface<
+    ::google::bigtable::v2::SampleRowKeysResponse>>
+LoggingDataClient::PrepareAsyncSampleRowKeys(
+    ::grpc::ClientContext* context,
+    const ::google::bigtable::v2::SampleRowKeysRequest& request,
```

Comment:
Oops. An accidental west-const here. As you know, we use east-const (`const int` == "west const"; `int const` == "east const").

(I wish there was a tool to fix this. There's discussion of teaching clang-format how to fix this, but that has been stalled for a couple years.)